packages/lesswrong/components/async/editor-plugins/draft-js-mathjax-plugin/src/mathjax/loadMathJax.js

```diff
@@ -5,7 +5,7 @@ import load from './load'
 const DEFAULT_SCRIPT = 'https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js'
 
 const DEFAULT_OPTIONS = {
-  loader: {load: ['[tex]/colorV2']},
+  loader: {load: ['[tex]/colorv2']},
```

Comment:
What's going on with the capitalization here? A quick search for similar example code turned up https://docs.mathjax.org/en/v3.0-latest/input/tex/extensions/colorV2.html which has it capitalized. Does un-capitalizing turn this into a different module, or a noop, or what?

Reply:
Indeed, I am similarly confused. The documentation used to say that you should capitalize it. And it used to work. But it stopped working at some point, and now it only works in the lower-case version, and the documentation has the lowercase version.

Reply:
It was renamed in between 3.0 and 3.1 (https://docs.mathjax.org/en/v3.0-latest/input/tex/extensions/colorV2.html vs https://docs.mathjax.org/en/v3.1-latest/input/tex/extensions/colorv2.html) and then maybe fixed so that both capitalizations work in 3.1.2 (https://github.com/mathjax/MathJax-src/pull/485)